src/improvements/template/SetGameImplementationTemplate.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
```

Comment:
When writing a template, you'll need to include an example task in `test/task/example`. See the docs where this is mentioned: https://github.com/ethereum-optimism/superchain-ops/blob/main/src/improvements/doc/NEW_TEMPLATE_GUIDE.md#additional-steps-when-creating-a-new-template

Similarly, you'll also need a test in `Regression.t.sol`.